### ISSUE TITLE
There are some  discrepancies between your paper and  code.

### ISSUE THREAD

**Issue by liunangithub** [ioangatop/srVAE#5]
``class DenseNetBlock(nn.Module):
    def __init__(self, inplanes, growth_rate, drop_prob=0.0):
        super().__init__()
        self.dense_block = ### ### nn.Sequential(
            Conv2d(inplanes, 4 * growth_rate,
                   kernel_size=1, stride=1, padding=0, drop_prob=drop_prob),
            Conv2d(4 * growth_rate, growth_rate,
                   kernel_size=3, stride=1, padding=1, drop_prob=drop_prob, act=None)
        )

    def forward(self, input):
        y = self.dense_block(input)
        y = torch.cat([input, y], dim=1)
        return y

### 

in  the part , I think it may miss a ELU function between two COnv2d models.Am I right?
